src/services/todo-lists-service.js

```diff
@@ -113,13 +138,49 @@ class TodoListsService {
                     })
             }
             else
+                Utils.log(`Could not update list ${id} to cluster. Will update local list`);
+
+                var todoListsResponse = nodesService.getAllButSelf().map(node => {
+                    return axios.get(`${Utils.getUrlForPort(node)}/lists/${id}`)
+                })
+
+                Promise.all(todoListsResponse)
+                    .then(todoListsResponse => {
+                        var todoLists = todoListsResponse.map(todoList => {return todoList.data})
+                        var quorumList = this.getQuorumList(todoLists)
+
+                        if (!quorumList)
+                            throw `No quorum achieved for list ${id}!`
+
+                        listRepository.updateToDoList(id, quorumList)
+                    })
+                    .catch(error => {
+                        Utils.log(`Error reading from quorum for list ${id}`, error.response.data)
+                    })
+
                 return {
                     isOk: false,
                     message: "Unable to update list because is being modified by another user"
                 }
         })
     }
 
+    requiredQuorum() {
+        return Math.floor(nodesService.get().length / 2)
+    }
+
+    getQuorumList(todoLists) {
+        var groupedToDoLists = Utils.groupBy(todoLists, "title")
```

Comment:
El groupby lo haces para agarrar todos los "read iguales"? Si es asi, creo que tendrias que agrupar por Hash, en vez de title. Not sure si entendi bien. @iruasi @isauri

Reply:
Entendiste bien, solo que cuando escribí esta función todavía no teníamos el hash implementado, por eso me agarré del título. Quise dejarlo un //TODO y me colgué

Ahora lo implemento con hash!